Tests/js/views/dashboard/assets/ez-dashboardblockallcontentview-tests.js

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockallcontentview-tests', function (Y) {
+    'use strict';
+
+    var renderTest,
+        getDataTest,
+        uiEventsTest,
+        CLASS_LOADING = 'is-loading',
+        CLASS_ROW_SELECTED = 'is-row-selected',
+        SELECTOR_CONTENT = '.ez-allcontent-block-content',
+        SELECTOR_ROW = '.ez-allcontent-block-row',
+        SELECTOR_OUTSIDE = '.outside',
+        VIEW_CONFIG = {container: '.container'};
+
+    renderTest = new Y.Test.Case({
+        name: 'eZ Dashboard All Content Block View render test',
+
+        setUp: function () {
+            this.view = new Y.eZ.DashboardBlockAllContentView(VIEW_CONFIG);
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+        },
+
+        'Should render view correctly with loading state': function () {
```

Comment:
*correctly* ? That's very inaccurate. This test should be called something like 'Should initially render in loading state'. Also the template variable are not tested.